console/console-init/ui/src/modules/project/containers/ProjectListContainer/ProjectListContainer.tsx

```diff
@@ -83,6 +88,15 @@ export const ProjectListContainer: React.FC<IProjectListContainerProps> = ({
     setDeleteIoTProjectQueryVariables
   ] = useMutationQuery(DELETE_IOT_PROJECT, ["allProjects"]);
 
+  var project_count = useQuery<IAllProjectsResponse>(RETURN_COUNT_PROJECTS(), {
+    fetchPolicy: FetchPolicy.NETWORK_ONLY,
+    pollInterval: POLL_INTERVAL
+  });
+
+  const totalProjects = (project_count && project_count.data) || {
```

Comment:
Can you please rename totalProjects to allProjects or something similar, as total project gives me an indication that it's the count of the projects.

Reply:
allProjects is already defined, how about projectsList?